### PR TITLE
Initial PR getting tests to pass

### PR DESCRIPTION
**Pull request by mattheww95**

### CODE REVIEW

**Comment by mattheww95** [comment]
This is an initial commit for for the cluster splitting pipeline, more work will need to be dont to deal with the actual goal the pipeline is supposed to address. 

Currently there appears to be some issues in how some files are being constructed e.g. column order changes. This may be due to how the `config` file specific to arborator is set up.

But this commit can serve as a good starting point for the rest of the pipeline.

File: .nf-core.yml [hunk 1]
```diff
@@ -15,9 +15,12 @@ lint:
     - .github/ISSUE_TEMPLATE/bug_report.yml
     - .github/PULL_REQUEST_TEMPLATE.md
     - .github/workflows/branch.yml
+    - .github/workflows/linting_comment.yml
```
Comment (apetkau) [inline .nf-core.yml:18]: Could the `linting_comment.yml` and `linting.yml` files not be added to the files to ignore linting and instead the files themselves updated to match what's found e.g., in here: https://github.com/phac-nml/snvphylnfc/tree/dev/.github/workflows

File: workflows/cluster_splitter.nf [hunk 2]
```diff
@@ -142,16 +142,16 @@ workflow CLUSTER_SPLITTER {
...
-    if (params.email || params.email_on_fail) {
-        NfcoreTemplate.email(workflow, params, summary_params, projectDir, log)
-    }
-    NfcoreTemplate.dump_parameters(workflow, params)
-    NfcoreTemplate.summary(workflow, params, log)
-    if (params.hook_url) {
-        NfcoreTemplate.IM_notification(workflow, params, summary_params, projectDir, log)
-    }
-}
+//workflow.onComplete {
```
Comment (apetkau) [inline workflows/cluster_splitter.nf:145]: Could you delete this instead of commenting this out?

**Review by apetkau — CHANGES_REQUESTED** [review]
Thanks so much @mattheww95. Sorry about the long wait for this PR. I just have 2 comments.

File: .nf-core.yml (hunk 1 above)
Comment (mattheww95) [reply .nf-core.yml:18]: Thank you for the example! I will take care of that.

File: workflows/cluster_splitter.nf (hunk 2 above)
Comment (mattheww95) [reply workflows/cluster_splitter.nf]: fixed in: https://github.com/phac-nml/clustersplitter/pull/1/commits/7b234b1a1fc8f330f895cca80afe115f4dff5981

File: .nf-core.yml (hunk 1 above)
Comment (mattheww95) [reply .nf-core.yml]: Tests are passing locally, but no in actions. Do you have any suggestions on what could resolve the issue?

**Pull request merged**